docs/running.md

```diff
@@ -154,6 +154,32 @@ The public key was spliced in as it was, in base64 format, which is how it would
 own genesis file format. Note that here we don't have the option to use `Address`, because we have to return
 these as actual `PublicKey` types to Tendermint through ABCI, not as a hash of a key.
 
+### (Optional) Add ipc to the Genesis file
+
+If you need ipc related function, let's add the subnet info to the Genesis with deployed subnet id: /r31415926
+
+```shell
+cargo run -p fendermint_app --release -- \
+      genesis --genesis-file test-network/genesis.json \
+      ipc \
+      gateway --subnet-id /r31415926 \
+      --bottom-up-check-period 10 --top-down-check-period 10 \
+      --msg-fee 1 --majority-percentage 60;
```

Comment:
```suggestion
      --msg-fee 1 --majority-percentage 65;
```
At least that number if higher than 2/3. With 60 I wonder if this is something other than BFT, does it tolerate 40% adversary, etc. But please correct me if I'm wrong and this has nothing to do with the classical restriction of Byzantine power.